rapids_build_backend/config.py

```diff
@@ -15,7 +15,9 @@ class Config:
         "allow-nightly-deps": (True, True),
```

Comment:
According to your current code this config var is no longer being used, but we do need some way to handle this configuration. I'm not sure what the best solution is yet though, we probably need to have a meeting and discuss our options.

Reply:
I'm thinking we could add a second configuration option, something like `nightly-dep-packages`, which is a list of packages which should have `>=0.0.0a0` added. This option could have the old `_VERSIONED_RAPIDS_WHEELS` as a sensible default.

Reply:
Yes, I've removed it.